kylin-spark-project/kylin-spark-engine/src/main/java/org/apache/kylin/engine/spark/job/NSparkExecutable.java

```diff
@@ -255,11 +255,13 @@ private ExecuteResult runSparkSubmit(KylinConfig config, String hadoopConf, Stri
 
             CliCommandExecutor exec = new CliCommandExecutor();
             Pair<Integer, String> result = exec.execute(cmd, patternedLogger, jobId);
+            updateMetaAfterBuilding(config);
+            //Add metrics information to execute result for JobMetricsFacade
```

Comment:
In which way do you "Add metrics information to execute result for JobMetricsFacade" ?